odoo/addons/base/tests/test_ormcache.py

```diff
@@ -23,7 +23,7 @@ def setUpClass(cls):
         # disabling autoretry to avoid hidding "real" errrors
         cls._retry = False
 
-    def test_ormcache(self):
+    def testormcache(self):
```

Comment:
Why?

Reply:
Just replacement with the logs caught this one too. Fixed.